lib/ast.js

```diff
@@ -1443,6 +1443,16 @@ var AST_Number = DEFNODE("Number", "value", {
     },
 }, AST_Constant);
 
+var AST_BigInt = DEFNODE("BigInt", "value", {
+    $documentation: "A BigInt literal",
+    $propdoc: {
+        value: "[string] the numeric representation",
+    },
+    _validate: function() {
+        if (typeof this.value != "string") throw new Error("value must be string");
```

Comment:
You could also add:
```js
if (this.value[0] == "-") throw new Error("bigint value can not be negative");
```

Reply:
Added `value` validation for both `AST_BigInt` & `AST_Number` in 773b7db869b6b6eae01c44d97866f96479051bc9